packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -64,12 +64,26 @@ export const useRootStyles = makeStyles({
       [thumbColorVar]: tokens.colorBrandBackgroundPressed,
       [progressColorVar]: tokens.colorBrandBackgroundPressed,
     },
+    '@media (forced-colors: active)': {
+      [railColorVar]: 'CanvasText',
+      [thumbColorVar]: 'Highlight',
+      [progressColorVar]: 'Highlight',
+      ':hover': {
```

Comment:
should this be `:hover, :active` to make sure both states' colors are overridden?

Reply:
I tested it and didn't see the need for it